packages/webdriverio/tests/commands/element/isStable.test.ts

```diff
@@ -0,0 +1,71 @@
+import path from 'node:path'
+import { expect, describe, it, beforeAll, beforeEach, afterEach, vi } from 'vitest'
+
+// @ts-ignore mocked (original defined in webdriver package)
+import got from 'got'
+import { remote } from '../../../src/index.js'
+import { ELEMENT_KEY } from '../../../src/constants.js'
+
+vi.mock('got')
+vi.mock('@wdio/logger', () => import(path.join(process.cwd(), '__mocks__', '@wdio/logger')))
+
+describe('isStable test', () => {
+    let browser: WebdriverIO.Browser
+    let elem: any
+
+    beforeAll(async () => {
+        browser = await remote({
+            baseUrl: 'http://foobar.com',
+            capabilities: {
+                browserName: 'foobar'
+            }
+        })
+        elem = await browser.$('#foo')
+    })
+
+    beforeEach(() => {
+        global.document = { visibilityState: 'visible' } as any
+    })
+
+    it('should allow to check if element is selected', async () => {
+        await elem.isStable()
+        expect(vi.mocked(got).mock.calls[2][0]!.pathname)
+            .toBe('/session/foobar-123/execute/async')
+        expect(vi.mocked(got).mock.calls[2][1]!.json.script)
+            .toBe(`return (function isElementStable(elem, done) {
+  if (document.visibilityState === "hidden") {
+    throw Error("You are are checking for animations on an inactive tab, animations do not run for inactive tabs");
+  }
+  try {
+    const previousPosition = elem.getBoundingClientRect();
+    requestAnimationFrame(() => {
+      requestAnimationFrame(() => {
+        const currentPosition = elem.getBoundingClientRect();
+        for (const prop in previousPosition) {
+          if (previousPosition[prop] !== currentPosition[prop]) {
+            done(false);
+          }
+        }
+        done(true);
+      });
+    });
+  } catch (error) {
+    done(false);
+  }
+}).apply(null, arguments)`)
```

Comment:
No need to assert the whole function
```suggestion
            .toBe(expect.stringContaining(`return (function isElementStable(elem, done)`))
```